### ISSUE TITLE
enable pagination for api calls

### ISSUE THREAD

**Issue by mike1703** [MISP/MISP#2465]
In case you have a well filled MISP instance and use for example the /attributes/restSearch/ endpoint with a query returning lots of information you might want to limit the number of results provided by one call.

The best way would be an optional pagination parameter to limit the amount of results returned.

**Comment by webdv** [comment]
Need this very badly, lot of timeouts

**Comment by adulau** [comment]
This has been implemented in the API (2.4.96 and upper). You can now do such query via restSearch (events or/and attributes level):

~~~~
{
    "returnFormat": "json",
    "last": "1d",
    "page": 1,
    "limit": 5
}
~~~~

Documentation is the following (available via MISP):

- `limit` Limit the number of results returned, depending on the scope (for example 10 attributes or 10 full events).
- `page` If a limit is set, sets the page to be returned. page 3, limit 100 will return records 201->300).
